### ISSUE TITLE
[BITV] 9.4.1.2/3.5 - "Details" section - The combobox for selecting "Tags" has not been implemented correctly, rendering it inaccessible for non-visual users who cannot understand how this element functions. For further implementation details, please refer to https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/. (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41893]
![image](https://github.com/nextcloud/server/assets/6078378/6c1a07c9-20ef-42fa-b632-8fa58efb66f0)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n5

9.4.1.2/11.5 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v11-n5
